Cue/app/tabs/library/LibraryHome.js

```diff
@@ -178,7 +178,7 @@ class LibraryHome extends React.Component {
     const MAX_LENGTH = 255
     CuePrompt.prompt(
       Platform.OS === 'android' ? 'Create new deck' : 'Create New Deck',
-      '',
+      'Enter a name for your new deck.',
```

Comment:
Nit: Saying "new deck" in both the title and the message seems repetitive. What if we just say "Enter a name for your deck"?